doc/source/cd/methods/mmddrift.ipynb

```diff
@@ -73,23 +73,22 @@
     "\n",
     "* `device`: *cuda* or *gpu* to use the GPU and *cpu* for the CPU. If the device is not specified, the detector will try to leverage the GPU if possible and otherwise fall back on CPU.\n",
     "\n",
+    "Additional KeOps keyword arguments:\n",
     "\n",
-    "Initialized drift detector example:\n",
+    "* `batch_size_permutations`: KeOps computes the `n_permutations` of the MMD^2 statistics in chunks of `batch_size_permutations`. Defaults to 1,000,000.\n",
+    "\n",
+    "Initialized drift detector examples for each of the available backends:\n",
     "\n",
     "\n",
     "```python\n",
     "from alibi_detect.cd import MMDDrift\n",
     "\n",
-    "cd = MMDDrift(x_ref, backend='tensorflow', p_val=.05)\n",
+    "cd_tf = MMDDrift(x_ref, backend='tensorflow', p_val=.05)\n",
+    "cd_torch = MMDDrift(x_ref, backend='pytorch', p_val=.05)\n",
+    "cd_keops = MMDDrift(x_ref, backend='keops', p_val=.05)\n",
     "```\n",
     "\n",
-    "The same detector in PyTorch:\n",
-    "\n",
-    "```python\n",
-    "cd = MMDDrift(x_ref, backend='pytorch', p_val=.05)\n",
-    "```\n",
-    "\n",
-    "We can also easily add preprocessing functions for both frameworks. The following example uses a randomly initialized image encoder in PyTorch:\n",
+    "We can also easily add preprocessing functions for the *TensorFlow * and *PyTorch* frameworks. The following example uses a randomly initialized image encoder in PyTorch:\n",
```

Comment:
Technically you could use any mixture of backends between the preprocessing and following drift detection computations, incl. backends which are not even explicitly supported by us. Of course I personally wouldn't encourage mixing these in practice since it can become painful for deployment, but we're not restricting anyone. I will update this sentence though.